### PR TITLE
Adjust report TlsCertificate table headers to match new TLS assets

### PR DESCRIPTION
**Pull request by swaterkamp**
**Checklist**:

- [X] Tests
- [X] [CHANGELOG](https://github.com/greenbone/gsa/blob/master/CHANGELOG.md) Entry

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -12,6 +12,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 - Display timezone for session timeout in user menu [#1764](https://github.com/greenbone/gsa/pull/1764)
 
 ### Changed
+- Changed report TlsCertifcate table headers to match TlsCertificate asstes
```
Comment (sarahd93) [inline CHANGELOG.md]:
```suggestion
- Changed report TlsCertificate table headers to match TlsCertificate assets
```

Comment (swaterkamp) [reply CHANGELOG.md]: Oops =0

File: CHANGELOG.md [hunk 2]
```diff
@@ -12,6 +12,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 - Display timezone for session timeout in user menu [#1764](https://github.com/greenbone/gsa/pull/1764)
 
 ### Changed
+- Changed report TlsCertificate table headers to match TlsCertificate assets
+table headers [#2042](https://github.com/greenbone/gsa/pull/2042)
```
Comment (sarahd93) [inline CHANGELOG.md]:
```suggestion
table headers [#2044](https://github.com/greenbone/gsa/pull/2044)
```

Comment (swaterkamp) [reply CHANGELOG.md]: Ok, now this seems like I had a small stroke...

**Pull request merged**